.gitignore

```diff
@@ -17,6 +17,8 @@
 .vs/
 .vscode
 Justfile
+*egg-info/
+towncrier.test.test_*/
```

Comment:
not sure why this is needed ... we can have it if it helps you. just asking :)

Reply:
I can remove it.
These dirs popped up after calling trial.

Reply:
I guess it was before me reading the CONTRIBUTOR docs. I tried running the tests with pytest.